snuba/admin/static/table.tsx

```diff
@@ -5,13 +5,15 @@ import { COLORS } from "./theme";
 type TableProps = {
   headerData: ReactNode[];
   rowData: ReactNode[][];
-  columnWidths: number[];
+  columnWidths?: number[];
 };
 
 function Table(props: TableProps) {
-  const { headerData, rowData, columnWidths } = props;
+  let { headerData, rowData, columnWidths } = props;
```

Comment:
nit: Why changing this? I normally prefer const unless reassigning later

Reply:
in my first version, i reassigned columnWidths. will update